### PR TITLE
Fix web-bot

### PR DESCRIPTION
**Pull request by EniasCailliau**
This PR makes the response format of the webbot compatible with what is currently supported by the front end.  

image

I also added the field `who` to ChatMessage so we wouldn't need a hacky way to include that field in our response (by default we ignore fields that aren't defined in the spec). For context, here's how the hackathon project added the who field: 

image
 

🚨 This PR temporarily removes support for source attribution. This will be added back in a follow-up PR to decide if and how sources fit into ChatMessage responses.

### CODE REVIEW

File: src/steamship/experimental/transports/chat.py [hunk 1]
```diff
@@ -6,6 +6,9 @@
 
 
 class ChatMessage(Block):
+
+    who: Optional[str] = "bot"
```
Comment (dkolas) [inline src/steamship/experimental/transports/chat.py:10]: Was this supposed to be in this pr?

File: src/steamship/experimental/package_starters/web_bot.py [hunk 2]
```diff
@@ -20,28 +29,13 @@ def create_response(self, incoming_message: ChatMessage) -> Optional[List[ChatMe
         raise NotImplementedError
 
     @post("answer", public=True)
-    def answer(self, **payload) -> Dict[str, Any]:
+    def answer(self, **payload) -> List[ChatMessage]:
```
Comment (dkolas) [inline src/steamship/experimental/package_starters/web_bot.py:32]: Sorry I botched this. Thanks for catching

**Review by dkolas — APPROVED** [review]
Looks good - there may be one stray unrelated line in the PR. Thanks for fixing.

File: src/steamship/experimental/transports/chat.py (hunk 1 above)
Comment (EniasCailliau) [reply src/steamship/experimental/transports/chat.py:10]: It was yes, without it the front-end thinks the message was written by the user.

**Pull request merged**